RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
```

Comment:
Minor? Do we expect anyone to have been using this function?

Reply:
no, I don't think anyone was using it :)

Reply:
I'm going to do minor just in case :D

Reply:
Might want to make the change clear in the RELEASE.md too? (i.e. "moved x from y to z")

Reply:
will do :)